app/src/pages/task-detail/task-detail.ts

```diff
@@ -25,6 +25,23 @@ export class TaskDetail {
     }
   }
 
+  peopleLeft(task){
```

Comment:
duplicated code? maybe this is not the best place for this method, how about the model?